driver-core/src/main/java/com/datastax/driver/core/Host.java

```diff
@@ -44,12 +44,12 @@
   // We use that internally because
   // that's the 'peer' in the 'System.peers' table and avoids querying the full peers table in
   // ControlConnection.refreshNodeInfo.
-  private volatile InetAddress broadcastAddress;
+  private volatile InetSocketAddress broadcastAddress;
```

Comment:
Could we rename this and the corresponding setter with `Socket`?
Right now we have:
* getBroadcastAddress returning an InetAddress
* setBroadcastAddress taking an InetSocketAddress

Reply:
I'll rename the setters to `*SocketAddress` for consistency.